static/src/javascripts/projects/common/modules/identity/email-sign-in-banner/template.js

```diff
@@ -15,12 +15,12 @@ export const make = (params: Params): string =>
             Enjoying ${params.campaign.name}?
          </h3>
          <p class="site-message--email-sign-in-banner-text">
-            Upsell copy
+            We want to help you find more newsleters you'll love. If you sign in we can suggest you newsletters tailored to your personal taste. And you can manage your preferences at any time.
```

Comment:
Like this copy a lot 👍 

Typo on `newsletters` though.

Reply:
will fix it but i think we are going with a different wording!